client/client.go

```diff
@@ -546,12 +534,10 @@ func (ec *SDKClient) TraceBlockByHash(
 }
 
 // TraceTransaction returns a Transaction trace
-func (ec *SDKClient) TraceTransaction(
-	ctx context.Context,
-	hash common.Hash,
-) (json.RawMessage, []*FlatCall, error) {
+func (ec *SDKClient) TraceTransaction(ctx context.Context, hash common.Hash) (json.RawMessage, []*FlatCall, error) {
```

Comment:
probably need to revert it. We use the previous formatting - it is more readable